backend-server/src/main/java/org/dice/ida/util/DbUtils.java

```diff
@@ -0,0 +1,9 @@
+package org.dice.ida.util;
+
+import org.dice.ida.constant.IDAConst;
+
+public class DbUtils {
+	public static String manageNullValues(String value) {
+		return (value == null || ValidatorUtil.isStringEmpty(value) || value.equals("?")) ? IDAConst.NULL_VALUE_IDENTIFIER : value;
```

Comment:
"?" should be in constants. All string literals that can be changed later on should be in constants